install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+if [[ -n $1 ]] && [[ $1 == '--scripts' ]]; then
+  echo '💡  Installing scripts…'
+
+  PATH_LINE='export PATH=$PATH:$HOME/.git-semantic-commits:$PATH'
+  BASHRC=~/.bashrc
+  ZSHRC=~/.zshrc
+
+  if [ -f $BASHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $BASHRC
+    then
+      echo
+      echo "📝  Adding path to $BASHRC"
+      echo $PATH_LINE >> $BASHRC
+      source $BASHRC
+    fi
+  fi
+
+  if [ -f $ZSHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $ZSHRC
+    then
+      echo
+      echo "📝  Adding path to $ZSHRC"
+      echo $PATH_LINE >> $ZSHRC
+      source $ZSHRC
+    fi
+  fi
+else
+  echo '💡  Installing git aliases…'
+
+  # git-extras/chore compatibility https://goo.gl/oZUN3V
```

Comment:
I don't mind having this in, but can you please post the full URL to the repo here?